mesheryctl/internal/cli/root/environments/create.go

```diff
@@ -35,7 +35,7 @@ var createEnvironmentCmd = &cobra.Command{
 	Long:  `Create a new environments by providing the name and description of the environment`,
 	Example: `
 // Create a new environment
-mesheryctl exp environment create --orgID [orgID] --name [name] --description [description] 
+mesheryctl exp environment create --orgId [orgId] --name [name] --description [description] 
```

Comment:
@MUzairS15 I don't know whether to laugh or cry.

Reply:
Case sensitivity....... 🙂

Reply:
> @MUzairS15 I don't know whether to laugh or cry.

I often find myself confused about certain issues, unsure whether they have already been resolved or are still being worked on. To help address this, I’ve made some reasonable assumptions in order to quickly handle completed issues. This way, contributors can better integrate into the community as soon as possible.  😂